lib/protocols.js

```diff
@@ -873,9 +874,14 @@
         if (keyMediaType.continue) {
           var valueMediaType = decodeServerMediaType(bytebuf);
           if (valueMediaType.continue)
+            var version=DECODE_UBYTE(bytebuf);
```

Comment:
If I get this correctly, this sets an assumption that protocol is 3.0 if we have more bytes

Reply:
Yeah this is making an assumption that protocol is >=3.0. but I guess here `version` is the maximum supported protocol version of the server.